e2e-tests/experiments/cstor-operator-provision/test_vars.yml

```diff
@@ -1,4 +1,4 @@
-cstor_operator_link: https://raw.githubusercontent.com/openebs/charts/gh-pages/{{ openebs_version }}/cstor-operator.yaml
+cstor_operator_link: https://raw.githubusercontent.com/openebs/charts/gh-pages/versioned/{{ openebs_version }}/cstor-operator.yaml
 release_operator_link: "https://raw.githubusercontent.com/openebs/charts/{{ commit_id }}/{{ openebs_version }}/cstor-operator.yaml"
```

Comment:
@nsathyaseelan -- should RC tags also be changed to include the `versioned` folder.

Reply:
no @kmova  for RC tags it will take the yaml's from PR we raised at that time of release. we were merging those PR's on the release date so we don't need to change that.